manifest.outlook.json

```
@@ -0,0 +1,145 @@
+{
+    "$schema": "https://developer.microsoft.com/json-schemas/teams/vDevPreview/MicrosoftTeams.schema.json",
+    "id": "f2b103f1-1ab1-4e1b-8f0b-072aa3d4e19d",
+    "manifestVersion": "devPreview",
+    "version": "1.0.0",
+    "name": {
+      "short": "Contoso Task Pane Add-in",
+      "full": "Contoso Task Pane Add-in"
+    },
+    "description": {
+      "short": "A template to get started.",
+      "full": "This is the template to get started."
+    },
+    "developer": {
+      "name": "Contoso",
+      "websiteUrl": "https://www.contoso.com",
+      "privacyUrl": "https://www.contoso.com/privacy",
+      "termsOfUseUrl": "https://www.contoso.com/servicesagreement"
+    },
+    "icons": {
+      "outline": "assets/outline.png",
+      "color": "assets/color.png"
+    },
+    "accentColor": "#230201",
+    "localizationInfo": {
+      "defaultLanguageTag": "en-us",
+      "additionalLanguages": []
+    },
+    "authorization": {
+      "permissions": {
+        "resourceSpecific": [
+          {
+            "name": "MailboxItem.Read.User",
+            "type": "Delegated"
+          }
+        ]
+      }
+    },
+    "validDomains": ["contoso.com"],
+    "extensions": [
+      {
+        "requirements": {
+          "scopes": ["mail"],
+          "capabilities": [
+            { "name": "Mailbox", "minVersion": "1.3" }
+          ]
+        },
+        "runtimes": [
+          {
+            "requirements": {
+              "capabilities": [
+                { "name": "Mailbox", "minVersion": "1.3" }
+              ]
+            },
+            "id": "TaskPaneRuntime",
+            "type": "general",
+            "code": {
+              "page": "https://localhost:3000/taskpane.html"
+            },
+            "lifetime": "short",
+            "actions": [
+              {
+                "id": "TaskPaneRuntimeShow",
+                "type":"openPage",
+                "pinnable": false,
+                "view": "dashboard"
+              }
+            ]
+          },
+          {
+            "id": "CommandsRuntime",
+            "type": "general",
+            "code": {
+              "page": "https://localhost:3000/commands.html",
+              "script": "https://localhost:3000/commands.js"
+            },
+            "lifetime": "short",
+            "actions": [
+              {
+                "id": "action",
+                "type": "executeFunction",
+                "displayName": "action"
+              }
```

Remove "displayName" property.
```suggestion
            "actions": [
              {
                "id": "action",
                "type": "executeFunction"
              }
```